cmd/main.go

```diff
@@ -49,14 +49,14 @@ var MainCmd = &cobra.Command{
 func init() {
 	err := config.Load()
 	if err != nil {
-		bail(fmt.Errorf("Failed to load configuration: %s", err))
+		bail(fmt.Errorf("failed to load configuration: %s", err))
 	}
 
 	prepareFlags()
 }
 
 func bail(err error) {
-	fmt.Fprintf(os.Stderr, "[Error] %s\n", err)
+	fmt.Fprintf(os.Stderr, "[Error] Stopping server: %s\n", err)
```

Comment:
Should we just use logrus everywhere now that we included it?

Reply:
Fine with me! It's only this one place but this is also where any fatal errors drop out.

7f55af4

This is the only place I used `Errorf`. Does that work for you?

I guess you know but logrus is limited to setting the output pointer globally. So everything is going to `stdout` currently.

Reply:
I think stdout is fine.